src/consensus/pbft/libbyz/Prepare.h

```diff
@@ -23,9 +23,16 @@ struct Prepare_rep : public Message_rep
   Seqno seqno;
   Digest digest;
   int id; // id of the replica that generated the message.
-
 #ifdef SIGN_BATCH
-  KeyPair::Signature batch_digest_signature;
+  size_t digest_sig_size;
+  PbftSignature batch_digest_signature;
+  static constexpr size_t padding_size =
+    ALIGNED_SIZE(pbft_max_signature_size) - pbft_max_signature_size;
+  std::array<uint8_t, padding_size> digest_padding;
+#endif
+
+#ifdef USE_PKEY
```

Comment:
What's USE_PKEY?

Reply:
Same as above

Reply:
So there's USE_PKEY_CHECKPOINTS and VIEW_CHANGES (which as far as I can tell are always on, and so in my opinion should be removed, because we never test without), but I can't find anything setting USE_PKEY or explaining what it's for, even though it's gating a fair bit of stuff in Prepare and Pre_Prepare. Can we also remove it?

Reply:
Sure created ticket for this: https://github.com/microsoft/CCF/issues/691

I want to remove these carefully to make sure nothing gets deleted by mistake

Feel free to move it to `experimental ePBFT` milestone if you think it's better there

Reply:
Thank you! I don't think it's a requirement for `experimental ePBFT1, I think it can be done later, along with #562 and so on.